ecommerce/views.py

```diff
@@ -8,6 +8,23 @@ class Home(View):
     def get(self,response):
         return render(response,'index.html')
 
+
+class Products(View):
+    def get(self,response):
+        return render(response, 'products.html')
+
+class SingleProduct(View):
+    def get(self,response)
+        return render(response, 'single-product.html')
+
+class Contact(View):
+     def get(self,response):
+        return render(request, 'contact.html')
+
+class About(View):
+     def get(self,response)
+        return render(request, 'about.html')
```

Comment:
first argument of render must be response

Reply:
Solved!